scripts/apps/search/controllers/MultiActionBarController.tsx

```diff
@@ -73,29 +73,13 @@ export function MultiActionBarController(
     };
 
     this.multiImageEdit = function() {
-        const originalImages = multi.getItems();
-        // load images fully. Using multi.getItems() doesn't work
-        // since it doesn't contain "subject" required for "usage terms"
```

Comment:
I think it's behaving as expected if I got it right, look [here](https://drive.google.com/file/d/13192EmxTgrPPQH1XOEcu6Hzjf2QRN8V6/view)

Reply:
That's strange because it doesn't work for me. Here are a [few](https://drive.google.com/open?id=1pGQSDJXHPh-dT1XSNu3dD2ATzXLVsY98) [more](https://drive.google.com/open?id=15p_Csjng-4V1w7HWz6D7Voi--_Y1vNCu) videos. I've merged your PR to develop and used a server of sd-master instance.

The first video displays that metadata is not persisted. The second one shows an unrelated bug that it's not possible to cancel editing metadata. Also, see the error in the console. See if you can reproduce and fix these. If not, fix image upload and I'll see how it works on the test instance of this PR.

Reply:
Not sure what's wrong in your side, it's working fine [here](https://drive.google.com/file/d/10uYM0QYE-nZAyA6jrX29f3rcnP9QnJ5x/view). I will update the PR soon with a fix for image upload.

Reply:
I forgot to mention that I was able to upload an image in my video without filling the required fields. Fix it as well if you can reproduce it. If not, we'll see how it works on the test instance.

Reply:
I checked it, and it's the issue with the server of sd-ansa, the `required` field, differ in `schema` and `validator` in case of sd-ansa. To me, it looks like a config issue. I asked Petr about the same and he says that it's not a big issue as long as user is not able to publish those images. I can't reproduce the same with other test instances.

Reply:
I don't know what was happening with fields not being persisted. I've tested it again and it works fine now.